pwa/src/components/entities/entitiesTable.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
-import { Table, Card, Spinner, Alert } from "@conductionnl/nl-design-system/lib";
-import { isLoggedIn } from "../../services/auth";
-import { Link } from "gatsby";
+import {Table, Card, Spinner, Alert} from "@conductionnl/nl-design-system/lib";
+import {isLoggedIn} from "../../services/auth";
+import {Link} from "gatsby";
```

Comment:
Hier is niks veranderd. Even kijken hoe je dit kan voorkomen.

Reply:
Hier is niets veranderd. alleen de layout van de code is veranderd. Komt dit doordat je een prettier aan hebt staan? Misschien goed om te kijken wat we hiermee kunnen doen.

Nu komen namelijk allemaal changes in git naar boven die zorgen voor ruis.